configuration_version.go

```diff
@@ -87,9 +86,9 @@ type ConfigurationVersion struct {
 // CVStatusTimestamps holds the timestamps for individual configuration version
 // statuses.
 type CVStatusTimestamps struct {
-	FinishedAt time.Time `json:"finished-at"`
-	QueuedAt   time.Time `json:"queued-at"`
-	StartedAt  time.Time `json:"started-at"`
+	FinishedAt string `jsonapi:"attr,finished-at"`
+	QueuedAt   string `jsonapi:"attr,queued-at"`
+	StartedAt  string `jsonapi:"attr,started-at"`
```

Comment:
This is waiting on a fix to be merged in the jsonapi package so that it can read other time formats.

Reply:
This is fixed [here](https://github.com/hashicorp/jsonapi/pull/1), will update this comment soon.